### PR TITLE
[MIN-49] Create load data from DVC version step

### PR DESCRIPTION
**Pull request by Christopher-Norman**
This PR adds the step to load in the validated data into the embedding pipeline from a specific DVC version using the generic step created in MIN 60.

### CODE REVIEW

File: pipelines/data_embedding_pipeline/data_embedding_pipeline.py [hunk 1]
```diff
@@ -12,9 +13,9 @@ def data_embedding_pipeline() -> None:
 
     Steps:
         fetch_data: A ZenML step which loads the data from a specified DVC data version.
-        embed_data: A ZenML step which embeds the text data into vectors and pushes to the vector database.
+        load_data: A ZenML step which embeds the text data into vectors and pushes to the vector database.
```
Comment (osw282) [inline pipelines/data_embedding_pipeline/data_embedding_pipeline.py]: I think there is a mistake here. Should `load_data` have the first description and `embed_data` have the second?

**Review by osw282 — CHANGES_REQUESTED** [review]
Just one mistake on the doc string, everything else looks good. 🔥

**Review by osw282 — APPROVED** [review]
LGTM! 🛸

**Review by KirsoppJ — APPROVED** [review]
LGTM

**Review by JonoCX — APPROVED** [review]

**Pull request merged**